File: snippets/node/connector-patch.ts
const { CohereClient } = require('cohere-ai');

const cohere = new CohereClient({
  token: '<<apiKey>>',
});

(async () => {
  const connector = await cohere.connectors.update(connector.id, {
    name: 'test-connector-renamed',
    description: 'A test connector renamed',
  });

  console.log(connector);
})();

File: snippets/java/app/src/main/java/ConnectorPatch.java
import com.cohere.api.Cohere;
import com.cohere.api.resources.connectors.requests.UpdateConnectorRequest;


public class ConnectorPatch {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().clientName("snippet").build();

        cohere.connectors().update("test-id", UpdateConnectorRequest.builder()
                .name("new name")
                .url("https://connector-example.com/search").build());
    }
}
